### DISCUSSION TITLE
userErrors: Store Ineligible for Shipping Functions

### DISCUSSION THREAD

**Discussion by horselditty** [Shopify/function-examples#486] [Feedback: Discounts APIs]
![image](https://github.com/Shopify/function-examples/assets/1610111/43a63e69-c09e-4b66-b836-f999682e12ad)

Any ideas why we are getting this error on a store?

**Comment by gil--** [comment]
Are you on checkout extensibility?

**Reply by horselditty** [reply]
Ahh, store was still using checkout.liquid.  
Thanks!
